web/packages/design/src/Icon/Icons.story.tsx

```diff
@@ -230,13 +229,7 @@ export const Icons = () => (
   </Flex>
 );
 
-const IconBox = ({
```

Comment:
This is an auto-generated file, I think what happened is in [this PR](https://github.com/gravitational/teleport/pull/47850/files#diff-e7feacc9c3d9e1b84c1508545d7f477c067c1b41524ac32d2bde79f6c937b4c6) the file was directly edited to add the type, however anytime someone runs the `process-icons` script to add a new icon, it will be overwritten, so we should've added the type to the `Icon/script.js` file instead. I'll do that in this PR.

Reply:
This bit me, so I fixed it here https://github.com/gravitational/teleport/pull/48573